docs/components/modeler/desktop-modeler/element-templates/using-camunda-connector-templates.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
I propose we move this section out of the element templates section. It is strictly "user documentation" and not related to element templates.